examples/gocean/eg7/README.md

```diff
@@ -0,0 +1,92 @@
+# PSyclone GOcean Example 7
+
+**Author:** J. Henrichs, Bureau of Meteorology
+
+## Introduction
+
+This is a simple test that shows how to use the read-only verification
+support in PSyclone. It is a stand-alone program that can be compiled
+and run. 
+
+## Compilation
+A makefile is provided to compile this example. If required
+it will compile the dl_esm_inf libraryW and the read-only verification
```

Comment:
spurious 'W'

Reply:
How on earth did my editor not flag this ... it is supposed to do syntax checking. Guess I need to figure out how to not ignore camel-case or so. Anyway, fixed.